inc/class-sc-config.php

```diff
@@ -130,12 +131,15 @@ private function _is_dir_writable( $dir ) {
 
 			if ( function_exists( 'fileowner' ) ) {
 				$wp_file_owner = @fileowner( __FILE__ );
+				// Pass in the temporary handle to determine the file owner.
 				$temp_file_owner = @fileowner( $temp_file_name );
 
+				// Close and remove the temporary file.
 				@fclose( $temp_handle );
-				@unlink( $temp_file_name );
+				@unlink( $temp_handle );
```

Comment:
@tlovett1 That would help wouldn't it! 😆 Sorry, late night PR while sleepy! Patched. 👍